### PR TITLE
add EZStaking for Tori network

### PR DESCRIPTION
**Pull request by keupsonite**

### CODE REVIEW

File: ezstaking/chains.json [hunk 1]
```diff
@@ -316,6 +316,15 @@
         "run_time": "every 5 minutes",
         "minimum_reward": 1000000
       }
+    },
+    {
+      "name": "tori",
```
Comment (tombeynon) [inline ezstaking/chains.json:321]:
```suggestion
      "name": "teritori",
```

**Review by tombeynon — CHANGES_REQUESTED** [review]

File: ezstaking/chains.json (hunk 1 above)
Comment (keupsonite) [reply ezstaking/chains.json]: Thank you!
Don't hesitate to accept the changes directly next time :p

Comment (tombeynon) [reply ezstaking/chains.json]: Haha I would but Github wouldn't let me - I think there might be a setting in your account that prevents it

**Review by tombeynon — APPROVED** [review]

File: ezstaking/chains.json (hunk 1 above)
Comment (keupsonite) [reply ezstaking/chains.json]: Oh right, I'll have to check this out, thanks again :)

**Pull request merged**